scripts/debug_tools/prepare_analyzer_cmd.py

```diff
@@ -64,7 +64,24 @@ def __init__(
 def prepare(analyzer_command_file, pathOptions):
     res = lib.change_paths(get_first_line_of_file(analyzer_command_file),
                            AnalyzerCommandPathModifier(pathOptions))
-    return res
+
+
+    if '-nobuiltininc' not in res:
+        return res
+
+    # Find Clang include path
+    clang_lib_path = os.path.dirname(pathOptions.clang) + '/../lib'
```

Comment:
Isn't there an option for clang like in gcc to get the builtin includes?

Reply:
Yes there is an option: `-print-resource-dir` , but that is only available in Clang >= 5.0 .

Reply:
I think we could call clang with `-print-resource-dir` and if that is not supported (which is very unlikely, since we use at least 5.0 for CTU) then fall back to `os.walk`.
See: https://github.com/Ericsson/codechecker/blob/master/libcodechecker/analyze/host_check.py#L64